Test/Unit/Controller/Order/ReceivedUrlTest.php

```diff
@@ -125,81 +126,91 @@ public function execute_HappyPath()
         $url->method('getUrl')
             ->willReturn(self::REDIRECT_URL);
 
-        $this->cartHelper->expects($this->once())
+        $cartHelper = $this->createMock(CartHelper::class);
```

Comment:
@njfaries What's the logic behind replacing $this->cartHelper with a local variable $cartHelper? And others below?

Reply:
I found that if I set a method return on $this->cartHelper in one test it would be the same for other tests as well. Switching the mocks that required a few different configurations to local variables seemed like the reasonable way to solve this problem, but it's possible there's a proper way to handle it that I've missed.